### PR TITLE
Obmin 326 create reset all button

### PR DESCRIPTION
**Pull request by Kyslytsia**
Create ResetAllButton

### CODE REVIEW

**Review by Wolshebnik — CHANGES_REQUESTED** [review]
wrong logic

File: package.json [hunk 1]
```diff
@@ -36,10 +36,12 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-input-mask": "^3.0.0-alpha.2",
+    "react-redux": "^8.0.5",
```
Comment (Wolshebnik) [inline package.json:39]: If you connect redux 
where do you use it?

File: package.json [hunk 2]
```diff
@@ -36,10 +36,12 @@
...
     "react-responsive": "^9.0.2",
     "react-router-dom": "^6.8.0",
     "react-toastify": "^9.1.1",
     "react-tooltip": "^5.8.2",
+    "redux-toolkit": "^1.1.2",
```
Comment (Wolshebnik) [inline package.json:44]: the same

File: src/components/reset-all-button/ResetAllButton.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+// ProductOwnerDataDefault.args = {
+//   city: 'Kyev',
+//   avatar: image,
+//   name: 'Natali',
+//   date: '26.02.2023',
+//   cityText: 'Місто:',
+//   phoneText: 'Телефон:',
+//   phone: '097 357 37 77',
+//   dateText: 'Дата публікації:',
+// };
```
Comment (Wolshebnik) [inline src/components/reset-all-button/ResetAllButton.stories.tsx:24]: why do you want commented code pushed in the main?

File: src/components/reset-all-button/index.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  setAge,
+  setGender,
+  setSeason,
+  setSize,
+  setLocationCurrent,
+  setPreViewImage,
+  setCurrentIndexImage,
+  setShowLocation,
+  clearAdv,
+  getText = 'Cancel Everything',
```
Comment (Wolshebnik) [inline src/components/reset-all-button/index.tsx:24]: why so many props?
and why not sorted?

File: src/components/reset-all-button/index.tsx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    setAge([]);
+    setGender([]);
+    setSeason([]);
+    setSize('');
+    setLocationCurrent(null);
+    setPreViewImage([]);
+    setCurrentIndexImage(null);
+    setShowLocation({ city: '', area: '' });
+    dispatch(clearAdv());
+  };
```
Comment (Wolshebnik) [inline src/components/reset-all-button/index.tsx:46]: Why so many set?

**Review by Wolshebnik — CHANGES_REQUESTED** [review]

File: src/components/cancel-everything/CancelEverything.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import { CancelEverything } from '.';
+import { argTypes } from './arg-types';
+
+export default {
+  title: 'CancelEverything',
+  component: CancelEverything,
+  argTypes: { ...argTypes, onClick: { action: 'onClick' } },
```
[the remark this answers is not in the document]
Comment (cloudmenson) [reply src/components/cancel-everything/CancelEverything.stories.tsx:9]: Get it out of here onClick: { action: 'onClick' }, describe it in argTypes

**Review by Viktoria-Serkiz — APPROVED** [review]

File: src/components/cancel-everything/index.tsx [hunk 7]
```diff
@@ -0,0 +1,13 @@
+import { IResetAllButton } from './types';
+import * as Styles from './styles';
```
Comment (cloudmenson) [inline src/components/cancel-everything/index.tsx:2]: sort pls

**Review by cloudmenson — APPROVED** [review]

**Review by Voloshkina — APPROVED** [review]

**Pull request merged**